src/Xamarin.Android.Build.Tasks/Tasks/CilStrip.cs

```diff
@@ -31,12 +31,7 @@ public CilStrip ()
 
 		public override bool RunTask ()
 		{
-			try {
-				return DoExecute ();
-			} catch (Exception e) {
-				Log.LogCodedError ("XA3001", "{0}", e);
-				return false;
-			}
```

Comment:
Should we remove this? What happens if we get an error which we are not expecting? This would then bubble up to MSBuild and give a horrible MSxxxx error. Should we keep a generic error here which asked users to report the issue ?

Reply:
Thanks to Pobst, it should go there here now: https://github.com/xamarin/xamarin-android/blob/1bee4ad176863bdfc0de258542a2c3685fc60f2d/src/Xamarin.Android.Build.Tasks/Tasks/AndroidTask.cs#L16-L21

I think using the default "let it throw" behavior is better now as long as this subclasses `AndroidTask`.